sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/SquareEditor.java

```diff
@@ -216,7 +219,11 @@ protected boolean doControlPointMoved(ControlPoint oCP, IGeoPosition oLatLon) {
                 currentBearing = (currentBearing + 360) % 360;
                 recompute(this.oFeature.getPosition());
                 this.oFeature.setAzimuth(currentBearing);
-                this.oFeature.apply();
+                try {
+                    storageManager.apply(mapInstance, oFeature, null);
+                } catch (EMP_Exception ex) {
+                    Log.e(TAG, "Apply failed.", ex);
```

Comment:
I am sure there is a good reason, but I am not sure why if it fails we swallow the exception. I noticed this is the same for all the editors.  Any idea why we chose to do it this way?

Reply:
This is in the draw editors. If the change fails to apply, the user will see it.